api/proto/common/v2/common_v2.proto

```diff
@@ -3,21 +3,37 @@ package common.v2;
 import "common/common.proto";
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common/v2";
 
-
+// BlobHeader contains the information describing a blob and the way it is to be dispersed.
 message BlobHeader {
   // Blob version
   uint32 version = 1;
+  // quorum_numbers is the list of quorum numbers that the blob is part of.
+  // All quorums must be specified (including required quorums).
+  //
+  // The following quorums are currently required:
+  // - 0: ETH
+  // - 1: EIGEN
   repeated uint32 quorum_numbers = 2;
+  // commitment is the KZG commitment to the blob
   common.BlobCommitment commitment = 3;
+  // payment_header contains payment information for the blob
   common.PaymentHeader payment_header = 4;
-
   // signature over keccak hash of the blob_header that can be verified by blob_header.account_id
   bytes signature = 5;
 }
 
-// BlobCertificate is what gets attested by the network
+// BlobCertificate contains a full description of a blob and how it is dispersed. Part of the certificate
+// is provided by the blob submitter (i.e. the blob header), and part is provided by the disperser (i.e. the relays).
+// Validator nodes eventually sign the blob certificate once they are in custody of the required chunks
+// (note that the signature is indirect; validators sign the hash of a Batch, which contains the blob certificate).
 message BlobCertificate {
```

Comment:
Since we are re-evaluating the naming, I think we may consider this name as well. It may be something like `TBSBlobCertificate`, TBS stands for "To Be Signed".

It's a practice seen in https://www.rfc-editor.org/rfc/rfc5280#section-4.1.1
Where a actual certificate is like:
```
Certificate ::= SEQUENCE {
    tbsCertificate          TBSCertificate,
    signatureAlgorithm      AlgorithmIdentifier,
    signature               BIT STRING
  }
```

Reply:
I've created a ticket to revisit this. I'd prefer not to rename any additional things as a part of this PR. https://linear.app/eigenlabs/issue/EGDA-808/rename-blobcertificate